### PR TITLE
Adding links for exploring ivvv in MyBinder [WIP]

### PR DESCRIPTION
**Pull request by kmader**
adding the code and links for playing around with the module in mybinder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # ivvv
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/kmader/ivvv/patch-2)
```
Comment (ghost) [inline README.md]: In order to merge this into imaging-tools:master shouldn't all the gh/kmader lines be changed to gh/imaging-tools ?

File: README.md [hunk 2]
```diff
@@ -18,6 +20,17 @@ Standard scientific python toolkit.
 ## How?
 We've used a stripped down version of 3D rendering provided by the [Allen Cell Explorer's](https://allencell.org) [3D Cell Viewer.](https://github.com/dmt-aics/volume-viewer)
 
+
+
+### Demo Notebooks
+
+- [Foam 3D](https://mybinder.org/v2/gh/kmader/ivvv/patch-2?filepath=examples%2Ffoam_3d.ipynb)
```
Comment (ghost) [inline README.md]: Same as above - in order to merge this into imaging-tools:master shouldn't all the gh/kmader lines be changed to gh/imaging-tools ?

Comment (kmader) [reply README.md]: Yes I didn't want to change them because the links then don't work (until the PR is accepted) but I can change them now

**Comment by toloudis** [comment]
Thank you for the contribution!

**Pull request merged**